### ISSUE TITLE
Macro Class condition won't work ?

### ISSUE THREAD

**Issue by playinter** [OpenKore/openkore#703]
i has same as this problem http://forums.openkore.com/viewtopic.php?f=32&t=212065

**Comment by Zexeta** [comment]
Yep the problem still exists.  I have had to make sloppy workaround macros that check base, job and equipped weapon in order to achieve the same result.  I hope this is fixed soon.

**Comment by alisonrag** [comment]
someone else can confirm? maybe another condition was locking the automacro

**Comment by AlexandreVitorino** [comment]
Maybe something about race condition.

**Comment by Nipodemos** [comment]
i think this can be already closed right?

**Comment by Fadreus** [comment]
He didn't give any feedback nor more information.
I've tested it with no problem.
Close and disclose if anyone else having this exact issue I guess. :thinking:
